app/assets/javascripts/discourse/app/components/modal/history.js

```diff
@@ -171,6 +172,9 @@ export default class History extends Component {
     Post.loadRevision(postId, postVersion).then((result) => {
       this.postRevision = result;
       this.loading = false;
+      if (this.initialPaint) {
+        this.initialPaint = false;
+      }
```

Comment:
For extra golf: 😉 

```suggestion
      this.initialPaint = false;
```

(and maybe `initialLoad` or something like that might be better, we don't _usually_ use "paint" in names)